packages/mobile/src/navigator/NavigatorWrapper.tsx

```diff
@@ -46,7 +51,21 @@ export const NavigatorWrapper = () => {
   const [isReady, setIsReady] = React.useState(RESTORE_STATE ? false : true)
   const [initialState, setInitialState] = React.useState()
   const appLocked = useTypedSelector(getAppLocked)
+  const minRequiredVersion = useTypedSelector((state) => state.app.minVersion)
   const routeNameRef = React.useRef()
+  const dispatch = useDispatch()
+
+  const updateRequired = React.useMemo(() => {
+    if (!minRequiredVersion) {
+      return false
+    }
+    const version = DeviceInfo.getVersion()
+    const versionBelowMinimum = isVersionBelowMinimum(version, minRequiredVersion)
+    if (!versionBelowMinimum) {
+      dispatch(minAppVersionDetermined(null))
+    }
+    return versionBelowMinimum
+  }, [minRequiredVersion])
```

Comment:
Still feels wrong, I don't understand why we need to call `dispatch` here.

This should be in a selector I think.

Reply:
Well, I just wanted to avoid doing this device check every time, if we already know the user updated the app I feel it would be better to have the `minRequiredVersion` as null but I guess we can just not dispatch anything and check against the latest minRequiredVersion found every time. 

Is something like this what you had in mind?

Reply:
Yes I think so! Happy to chat about this tomorrow, I might have missed something here.